src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -224,12 +224,12 @@ internal static string ShortenCategoryName(string fullCategoryName)
             if (string.IsNullOrEmpty(fullCategoryName))
                 return string.Empty;
 
-            var catName = fullCategoryName.Replace(Configurations.CategoryDelimiter.ToString(), " " + Configurations.ShortenedCategoryDelimiter + " ");
+            var catName = fullCategoryName.Replace(Configurations.CategoryDelimiterString.ToString(), " " + Configurations.ShortenedCategoryDelimiterString + " ");
 
             // if the category name is too long, we strip off the interior categories
             if (catName.Length > 50)
             {
-                var s = catName.Split(Configurations.ShortenedCategoryDelimiter).Select(x => x.Trim()).ToList();
+                var s = catName.Split(Configurations.ShortenedCategoryDelimiterString.ToCharArray()).Select(x => x.Trim()).ToList();
```

Comment:
Can we not keep `ShortenedCategoryDelimiter` (which is a `char`) instead of doing this `ToCharArray` method call (which is more expensive)?